components/Table.tsx

```diff
@@ -265,6 +270,19 @@ export const Table: React.FC<Props> = ({ data, hasFilters = true }) => {
 
   const router = useRouter();
 
+  function onClickRow(row: TRow<Synth<{ type: ContractType }>>) {
+    if (row.original.externalUrl) {
+      const newWindow = window.open(
+        row.original.externalUrl,
+        "_blank",
+        "noopener,noreferrer"
+      );
+      if (newWindow) newWindow.opener = null;
+    } else {
+      router.push(`/${row.original.address}`);
+    }
```

Comment:
If the new Contentful field `externalUrl` is populated, then open it in a new tab